### PR TITLE
53: remove mean damage data types for which we don't have damage ratios

### PR DESCRIPTION
**Pull request by vnarah2**
This task shouldn't impact anything. 
Non-structural mean damage is probably broken because the damage ratio datasets related to AS/DS are still using slight, moderate etc. instead of DS_X convention - This task doesn't fix it. It will be fixed in #52

### CODE REVIEW

**Comment by navarroc** [comment]
NS damage I think works because we pass in the "keys" to pull the DS probabilities and in the meandamage code, we never assume row1 is Insignificant, we never actually parse the DS column from the damage ratios.

**Comment by navarroc** [comment]
I just tested the mean_dmg.ipynb with the latest release and it works for non structural.

**Review by longshuicy — APPROVED** [review]

**Comment by navarroc** [comment]
Can you fix the mean_damage method documentation? It says "Calculates building damgae results for a single building", this should be more generic, something like "Calculates mean damage based on damage probabilities and damage ratios"

**Comment by vnarah2** [comment]
> Can you fix the mean_damage method documentation? It says "Calculates building damgae results for a single building", this should be more generic, something like "Calculates mean damage based on damage probabilities and damage ratios"

@navarroc Done

File: pyincore/analyses/meandamage/meandamage.py [hunk 1]
```diff
@@ -173,10 +173,10 @@ def mean_damage_bulk_input(self, damage, dmg_ratio_tbl):
         return result
 
     def mean_damage(self, dmg, dmg_ratio_tbl, damage_interval_keys, is_bridge):
-        """Calculates building damage results for a single building.
+        """Calculates mean damage based on damage probabilities and damage ratios
 
         Args:
-            dmg (obj): dmg analysis output for a single entry.
+            dmg (obj): dmg analysis output for a single building.
```
Comment (navarroc) [inline pyincore/analyses/meandamage/meandamage.py:179]: This is generic so it shouldn't say for a building. Maybe "damage analysis output for a single entity in the built environment"?

Comment (vnarah2) [reply pyincore/analyses/meandamage/meandamage.py]: ah right! fixed. thanks for pointing it out

**Review by navarroc — APPROVED** [review]

**Pull request merged**